README.md

```diff
@@ -106,7 +106,7 @@ Table of contents:
 * [Built With Angular 2](http://builtwithangular2.com/)
 * [`#ng-2 Slack Channel`](https://angularbuddies.slack.com/messages/ng-2/) on AngularBuddies ([Sign up](http://www.angularbuddies.com/))
 * [`#angular2 Slack Channel`](https://dartlang.slack.com/messages/angular2) on dartlang.slack.com ([Sign up](https://dartlang-slack.herokuapp.com/))
-
+s
```

Comment:
Oh sorry it's an typo! I guess I did cmd + s thinking I was writing in a file. I created a PR via web. Sorry for it!!